docs/migration/index.md

```diff
@@ -0,0 +1,25 @@
+# Important Announcement: HPC Cluster Migration from Lochness to Wulver
+We are excited to share important news about the upcoming migration of users from the old HPC cluster, [Lochness](lochness.md), to the new and improved cluster, [Wulver](wulver.md). This migration is scheduled to commence on 1/16/2024.
+
+## Migration Details:
+
+* Start Date: January 16, 2024
+* Priority:  Migration will be carried out in PI groups, with PIs who own nodes in Lochness being migrated first.
+* Communication: PIs will receive prior communication from our team to discuss specific details tailored to their groups.
+* Lochness Complete Shutdown: At the conclusion of the migration, Lochness will undergo a complete shutdown to facilitate its merger with Wulver. You will be advised of any necessary preparations needed on your end for a seamless transition.
+* Restrictions on Lochness: As of the migration start date, no new user accounts or software installations will be permitted on Lochness. We appreciate your cooperation in adhering to these restrictions to ensure a smooth migration process.
```

Comment:
We made this change just before the email went out. Please correct to maintain consistency
Restrictions on Lochness: Effective immediately,